File: Solutions/2482. Difference Between Ones and Zeros in Row and Column/2482.java
class Solution {
    public int[][] onesMinusZeros(int[][] mat) {
        int m = mat.length;
        int n = mat[0].length;
        int[] rowSum = new int[m];
        int[] colSum = new int[n];
        for(int i=0;i<m;i++){
            for(int j=0;j<n;j++){
                if(mat[i][j] == 1){
                    rowSum[i]++;
                    colSum[j]++;
                }
            }
        }
        int[][] ans = new int[m][n];
        for(int i=0;i<m;i++){
            for(int j=0;j<n;j++){
                int onesRow = rowSum[i];
                int onesCol = colSum[j];
                ans[i][j] = onesRow + onesCol + (onesRow - m) + (onesCol - n);
            }
        }
        return ans;
    }
}

File: Solutions/2482. Difference Between Ones and Zeros in Row and Column/2482.py
class Solution:
    def onesMinusZeros(self, mat: List[List[int]]) -> List[List[int]]:
        m = len(mat)
        n = len(mat[0])
        row_sum = [0] * m
        column_sum = [0] * n
        for i in range(m):
            for j in range(n):
                if mat [i][j] == 1:
                    row_sum[i] += 1
                    column_sum[j] += 1
        ans = [[0] * n for _ in range(m)]
        for i in range(m):
            for j in range(n):
                onesRow = row_sum[i]
                onesCol = column_sum[j]
                ans[i][j] = onesRow + onesCol + (onesRow - m) + (onesCol - n)
        return ans